src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +34,33 @@
               className="modal-card-title has-text-weight-medium"
```

Comment:
It's a good practice to use the 'classnames' library for defining classes conditionally. It helps to keep your code clean and easy to read.